# Problem 2.15

In [1]:
import numpy as np
import scipy.special
import sympy as sp

$$
\psi_0 = \sqrt{\frac{m \omega}{\hbar}} \exp \left( -\frac{m\omega}{2 \hbar} x^2 \right)
$$

$$
P \left( |x| > \sqrt{\frac{2E}{m \omega^2}} \right) = P \left( |x| > \sqrt{\frac{\hbar}{m\omega}} \right)
$$

Let

$$
\xi \equiv \sqrt{\frac{m \omega}{\hbar}} x
$$

Now, because the function is even,

$$
\begin{eqnarray}
P \left( |x| > \sqrt{\frac{2E}{m \omega^2}} \right) & = & 2 \int_{\sqrt{\frac{\hbar}{m\omega}}}^{\infty} \sqrt{\frac{m\omega}{\pi\hbar}} \exp \left(-\frac{m\omega}{\hbar} x^2 \right) \, dx \\
& = & 2 \int_1^{\infty} \frac{1}{\sqrt{\pi}} e^{-\xi^2} \, d\xi
\end{eqnarray}
$$

In [2]:
xi = sp.symbols('xi')

In [5]:
sp.simplify(2 * sp.integrate(1 / sp.sqrt(sp.pi) * sp.exp(-xi**2), (xi, 1, sp.oo)))

1 - erf(1)

In [6]:
1 - scipy.special.erf(1)

0.15729920705028522

So there is about a 15.7% chance that a particle in the ground state of the harmonic oscillator will be found in a classically forbidden region..